archey/entries/wan_ip.py

```diff
@@ -2,71 +2,74 @@
 
 from socket import timeout as SocketTimeoutError
 from subprocess import check_output, DEVNULL, TimeoutExpired, CalledProcessError
-from urllib.error import URLError
+from urllib.error import HTTPError, URLError
 from urllib.request import urlopen
 
-from archey.configuration import Configuration
+from archey.entry import Entry
 
 
-class WanIp:
+class WanIp(Entry):
```

Comment:
Sorry for these noises, but I've thought it was a great opportunity to clean this class too...